docs/puml/monte_carlo/power_calculation.puml

```diff
@@ -0,0 +1,16 @@
+	
+@startuml
+
+start
```

Comment:
Maybe this is a bit too deep and content of another discussion. But, wouldn't it be sensible to map starting time to time series and join / iron / concatenate the time series in a very last step. Why? Because then we have the information which appliance is switched on for the longest period. Just if in future anyone might want to use this information.

Reply:
This was just to document the process on a very high level. I agree and suggest that we adapt it accordingly once we have a more concrete implementation